modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/Replace.java

```diff
@@ -50,17 +51,18 @@ public void setItems(List<Item> items) {
 
 	public void buildReplaceElement(Document doc, Element rootElement) {
 		if (getItems() != null) {
-			Element get = doc.createElement(Constants.REPLACE);
-			rootElement.appendChild(get);
+			Element replace = doc.createElement(Constants.REPLACE);
+			rootElement.appendChild(replace);
 			if (getCommandId() != -1) {
 				Element commandId = doc.createElement(Constants.COMMAND_ID);
 				commandId.appendChild(doc.createTextNode(String.valueOf(getCommandId())));
-				get.appendChild(commandId);
+				replace.appendChild(commandId);
 			}
 			if (getItems() != null) {
-				for (int x = 0; x < getItems().size(); x++) {
-					if (getItems().get(x) != null) {
-						getItems().get(x).buildItemElement(doc, get);
+				for (Iterator<Item> itemIterator = getItems().iterator(); itemIterator.hasNext(); ) {
```

Comment:
Unnecessary white space right at the end of the line just before the closing bracket. Please remove.